src/arch/x86_64/kernel/mod.rs

```diff
@@ -82,6 +82,10 @@ pub fn get_mbinfo() -> VirtAddr {
 	}
 }
 
+pub fn get_fdt() -> usize {
+	boot_info().hardware_info.device_tree.unwrap().get() as usize
+}
```

Comment:
This needs to return an `Option`, since this is not available on Uhyve (hence the CI failure).